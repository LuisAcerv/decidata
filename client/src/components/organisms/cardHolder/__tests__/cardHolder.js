import React from "react";
import { shallow } from "enzyme";
import CardHolder from "../";

describe("CardHolder", () => {
  it("should render correctly", () => {
    let wrapper;
    wrapper = shallow(<CardHolder />);
    expect(wrapper).toMatchSnapshot();
  });
});
