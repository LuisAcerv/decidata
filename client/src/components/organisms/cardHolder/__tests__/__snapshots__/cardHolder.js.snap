// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardHolder should render correctly 1`] = `ShallowWrapper {}`;
